### DISCUSSION TITLE
precalentamiento auto nivelacion

### DISCUSSION THREAD

**Discussion by bombardeo38** [mriscoc/Ender3V2S1#130] [Ideas]
lo primero es darte las gracias no e tenido tiempo de trastear con tu version me gustaria saber si se le puede añadir la funcion de precalentamiento para la nivelacion tanto de las esquinas como de la nibelacion por bl/cr automatica y una consulta hay alguna manera de mantener el final de carrerqa en z porsi falla el bl/cr para evitar que golpe la cama o añair la funcion de parada si eso sucede gracias por tus aportes

**Comment by mriscoc** [comment]
Hola, feliz de que nuestro trabajo te sea util. El precalentamiento automático sólo está disponible para la creación de malla. Precalentar para nivelar las esquinas no es estrictamente necesario por lo cual no es un procedimiento que sea muy extendido, pero es fácil agregar el precalentamiento a tus accesos rápidos junto con el "bed tramming" así que serían un par de clicks de distancia. Para este firmware no es necesario que quites el fin de carrera de Z, pero no pueden funcionar ambos (Bltouch y fin de carrera) a la vez.

**Comment by bombardeo38** [comment]
gracias por contestar una consulta limito el ventilador en el laminador pero aun asi se sube a 200 informacion de pantalla y los puentes o boladizos se enfrian en exceso y hay una mala ahderencia entre capas posibilidad de añadir un menu o limitacion a esto gracias

**Reply by mriscoc** [reply]
El firmware sigue las instrucciones colocadas en el G-code, es probable que luego de tu ajuste manual haya una instrucción que vuelva a establecer el valor para el ventilador de capa.
